content/api/commands/end.md

```diff
@@ -26,8 +26,7 @@ cy.end() // Does not make sense to chain off 'cy'
 
 ### Yields [<Icon name="question-circle"/>](/guides/core-concepts/introduction-to-cypress#Subject-Management)
 
-<List><li>`.end()` yields `null`.</li><li>`.end()` cannot be chained
-further.</li></List>
+- `.end()` yields `null`.
```

Comment:
Is end a query or a command?

Reply:
Good catch, it's a query. Moving it to the proper section of the sidebar/TOC.

Reply:
https://github.com/cypress-io/cypress/pull/24681